testdata/deployment/extensions/deployment.yaml

```diff
@@ -1,8 +1,11 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
```

Comment:
@zhouying7780 could you  please help provide a description on why is this change? Thanks !!

Reply:
@kasturinarra  The older api version has discarded , when use it will hit error: no matches for kind "Deployment" in version "extensions/v1beta1"